client/actions/planning/api.js

```diff
@@ -969,21 +969,39 @@ const markPlanningPostponed = (plan, reason) => ({
 
 /**
  * Export selected planning items as a new article
+ *
+ * First opens a modal where user can sort those and
+ * then it sends it to server.
  */
 function exportAsArticle() {
     return (dispatch, getState, { api, notify, gettext }) => {
         const state = getState()
 
-        return api.save('planning_export', {
-            desk: state.workspace.currentDeskId,
-            items: state.planning.selectedItems,
-        })
-        .then(() => {
-            notify.success(gettext('Article was created'))
-            dispatch(actions.planning.ui.deselectAll())
-        }, () => {
-            notify.error(gettext('There was an error when exporting'))
-        })
+        const sortableItems = state.planning.selectedItems.map((id) => ({
+            id,
+            label: state.planning.plannings[id].headline,
```

Comment:
Headline is removed from planning form. So, the label should point to description probably as it is empty.

Reply:
thx for notice, I've added a fallback to slugline and description if headline is not there